Classes/Private/Manager/Database/OCTDBMessageCall.h

```diff
@@ -0,0 +1,18 @@
+//
+//  OCTDBMessageCall.h
+//  objcTox
+//
+//  Created by Chuong Vu on 5/14/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "RLMObject.h"
+#import "OCTMessageCall.h"
+
+@interface OCTDBMessageCall : RLMObject
+
+@property NSDate *endTime;
```

Comment:
I'm wondering if maybe I should remove the endTime entirely from the OCTMessageCall and instead just have a property for NSTimeInterval. Whoever creates the OCTMessageCall will be responsible then for updating the NSTimeInterval when the call terminates. I think that would work better?

That way we can do away with the endTime property and instead just have NSTimeInterval as a property. What do you think?